### PR TITLE
merge cp-lji branch to develop

### PR DESCRIPTION
**Pull request by petrjasek**

### CODE REVIEW

**Comment by petrjasek** [comment]
@MarkLark86 I will go over it but if you could also scan it for any nonsense that would be great

File: assets/utils.js [hunk 1]
```diff
@@ -17,6 +17,16 @@ import {
     TO_BE_CONFIRMED_TEXT
 } from './agenda/utils';
 
+/*
+ * Import and load all locales that will be used in moment.js
+ * This should match the LANGUAGES defined in settings.py
+ *
+ * 'en' comes by default
+ */
+import 'moment/locale/fr-ca';
```
Comment (MarkLark86) [inline assets/utils.js:26]: This would need to be improved at some point, to use configured languages

File: newsroom/templates/download_item.txt [hunk 2]
```diff
@@ -1,11 +1,7 @@
 {{ item.slugline }}
 {{ item.headline }}
+{{ item.source }}
```
Comment (MarkLark86) [inline newsroom/templates/download_item.txt:3]: This changes the default format of text downloads. Might need this modification in the customer's theme folder instead

**Comment by MarkLark86** [comment]
@petrjasek I went over the changes. Not sure if I caught everything, but it mostly looked fine

**Pull request merged**